src/payment/strategies/checkoutcom/checkoutcom-payment-strategy.ts

```diff
@@ -0,0 +1,120 @@
+import { CheckoutActionCreator, CheckoutStore, InternalCheckoutSelectors } from '../../../checkout';
```

Comment:
This strategy seems very specific for `Checkout.com's GooglePay`, but the name implies that it is for checkout.com in general, I believe it would be better renamed to something else to avoid confusion. cc @mauricio-sg

Reply:
Renamed to CheckoutcomGooglepayPaymentStrategy to clarify intended behavior